sdk/communication/Azure.Communication.CallingServer/tests/Infrastructure/CallingServerTestEnvironment.cs

```diff
@@ -12,10 +12,12 @@ namespace Azure.Communication.CallingServer.Tests
     /// </summary>
     public class CallingServerTestEnvironment : CommunicationTestEnvironment
     {
+        public const string AlternateCallerId = "ALTERNATE_CALLERID";
```

Comment:
@minnieliu @RezaJooyandeh 
Should we move this to [CommunicationTestEnvironment.cs](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/communication/Azure.Communication.Common/tests/CommunicationTestEnvironment.cs) and create a Variable in the ACS Live Test KV?

Reply:
We usually only add it to CommunicationTestEnvironment for variables that will be shared between all the Communication SDKs. I think it is fine to keep it in CallingServerTestEnvironment